### PR TITLE
Add Genome and Proteomics labs

### PR DESCRIPTION
**Pull request by neoformit**
Two new Galaxy Labs (subdomains) that I've migrated from Galaxy AU.

The Lab pages can be seen rendered here:

https://labs.usegalaxy.org.au/?content_root=https://github.com/neoformit/galaxy_codex/blob/add-labs/communities/genome/lab/base.yml

https://labs.usegalaxy.org.au/?content_root=https://github.com/neoformit/galaxy_codex/blob/add-labs/communities/proteomics/lab/base.yml

Note that after merging this PR, these URLs will change to:

https://labs.usegalaxy.org.au/?content_root=https://github.com/galaxyproject/galaxy_codex/blob/main/communities/genome/lab/base.yml

https://labs.usegalaxy.org.au/?content_root=https://github.com/galaxyproject/galaxy_codex/blob/main/communities/proteomics/lab/base.yml

I have also updated the single cell lab README and renamed `communities/spoc/subdomain` to `communities/spoc/lab`, therefore the final "resting place" for the Single-Cell lab after merging this will be:

https://labs.usegalaxy.org.au/?content_root=https://github.com/galaxyproject/galaxy_codex/blob/main/communities/spoc/lab/base.yml

### CODE REVIEW

**Comment by neoformit** [comment]
@paulzierep

**Comment by neoformit** [comment]
Will need revision after #169 merges

**Comment by neoformit** [comment]
Have refactored to match the new structure @paulzierep

**Comment by neoformit** [comment]
@bebatut

File: labs/assets/site-switcher.png
Comment (bebatut) [inline labs/assets/site-switcher.png:1]: There are some files left the `labs` folder. Could they move?

Comment (neoformit) [reply labs/assets/site-switcher.png:1]: The labs/assets folder is for global assets that are used by multiple labs, so it doesn't make sense for them to live in any particular lab. We also agreed that we could have a top-level `labs` folder with links to all existing labs, so the labs are easier to find.

Comment (bebatut) [reply labs/assets/site-switcher.png:1]: > The labs/assets folder is for global assets that are used by multiple labs, so it doesn't make sense for them to live in any particular lab. 

The labs/assets folder could go in [`sources` folder](https://github.com/galaxyproject/galaxy_codex/tree/main/sources/data)` like the HTML templates for pages

> We also agreed that we could have a top-level `labs` folder with links to all existing labs, so the labs are easier to find.

Links to all existing labs would be good in `communities/all` we store resources shared by different communities. What do you think?

Comment (neoformit) [reply labs/assets/site-switcher.png:1]: >Links to all existing labs would be good in communities/all

Ok, how about `communities/all/labs/`? Could I just put the `assets` in there too, so the labs stuff is all in one place? That makes a lot more sense to me than mixing the "labs" and "tool usage" files together. Plus, these are assets for "all communities' labs" so I think the path name is logical.

Comment (neoformit) [reply labs/assets/site-switcher.png:1]: Done

**Review by bebatut — APPROVED** [review]

**Pull request merged**